nixos/modules/services/web-apps/miniflux.nix

```diff
@@ -45,6 +45,9 @@ in
           Configuration for Miniflux, refer to
           <link xlink:href="https://miniflux.app/docs/configuration.html"/>
           for documentation on the supported values.
+
+          Correct configuration for the database is already provided.
+          By default, listens on localhost:8080.
```

Comment:
> Just my thoughts, but I prefer to read nix sources as the best documentation.


This is precisely what I want to avoid. Documentation should be good enough not to have to read the source.

Reply:
I added a let binding to prevent the default and the docstring to diverge.

Reply:
> Documentation should be good enough not to have to read the source.

This. x 1000. We can't let our existing knowledge of nix keep newcomers out. Better documentation is key.